docs/docs/cli/_category_.json

```diff
@@ -1,9 +1,8 @@
 {
   "label": "Command Line Interface",
   "position": 5,
-  "collapsed": true,
   "link": {
-    "type": "generated-index",
-    "description": "How to use the command line interface (CLI)."
+    "type": "doc",
+    "id": "introduction"
```

Comment:
I found a problem that if I kept both `_category_.json` and `index.mdx` it would create a CLI page section inside the CLI section so you would be able to loop there. I fixed it via removing the `_category_.json` file and adding the required information in `index.mdx`. I followed the idea that's in the `index.mdx` at the docs start (the welcome file).

Let me know if that's ok for the automatic migration. Otherwise, I would need some help here to get it done.